### ISSUE TITLE
Install fails with "error: namespaces "jx" not found"

### ISSUE THREAD

**Issue by allsuper** [jenkins-x/jx#900]
Hello! Minikube as well as GKE Installs fail with the following error, that the jx namespace is not found. 

```...
Cloning the Jenkins X cloud environments repo to /root/.jx/cloud-environments
Counting objects: 545, done.
Total 545 (delta 0), reused 0 (delta 0), pack-reused 545
error creating cluster namespaces "jx" not found
error: namespaces "jx" not found
```
On minkube and GKE I managed to get round with using --namespace default flag. 

So it looks like the installer does not create the jx namespace correctly.

**Comment by rawlingsj** [comment]
Yeah we've noticed this starting to happen today.  Not too sure why yet but we have a workaround which is being released at the moment v1.2.83.  It should be available to download in ~20 mins.

**Comment by allsuper** [comment]
Thanks very much for the fast reply and especially for the fix. I will test it tomorrow.
